bliss/encoder/variational_dist.py

```diff
@@ -11,11 +11,12 @@
 
 
 class VariationalDistSpec(torch.nn.Module):
-    def __init__(self, survey_bands, tile_slen):
+    def __init__(self, survey_bands, tile_slen, include_galaxy_property_loss: bool = True):
         super().__init__()
 
         self.survey_bands = survey_bands
         self.tile_slen = tile_slen
+        self.include_galaxy_property_loss = include_galaxy_property_loss
```

Comment:
If `include_galaxy_property_loss` if false, if would be better to omit galaxy-specific variables from `self.factor_specs`.

Reply:
if we remove the galaxy-specific variables, it may require us to modify the neural network architecture. In `_parse_factors` function, we parse the input according to `self.factor_specs`.

Reply:
The network architecture should automatically adapt if you remove them. The method `n_params_per_source` counts the number of parameters for whatever factors are present in `self.factor_specs`, and adjusts the output dimension of the network accordingly.

Reply:
professor, could we have a face-to-face discussion about this?